### PR TITLE
add monitor pipeline

### PR DESCRIPTION
**Pull request by aguilinger**
Pull request into a branch just note.

You can see example metrics at https://grafana.aptible.com/d/lfoy8SqMz/fluentd-drain-testing?orgId=1&from=1621531708727&to=1621542237110

### CODE REVIEW

**Comment by aguilinger** [comment]
@almathew I also don't have permissions to this repo

**Comment by almathew** [comment]
@aguilinger you do now

File: Dockerfile.erb [hunk 1]
```diff
@@ -49,7 +50,5 @@ VOLUME ["/tmp/certs"]
 
 # Lumberjack
 EXPOSE 5000
-# Redis (used if the drain is a tail)
```
Comment (mdelaossa) [inline Dockerfile.erb:52]: thanks for cleaning this up

**Review by mdelaossa — APPROVED** [review]
Nice, this looks nice and simple (but I bet it was hard to figure out initially)
LGTM

**Pull request merged**